samples/nrf9160/modem_shell/src/modem_trace/modem_trace_memfault.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <stdint.h>
+#include <memfault/metrics/metrics.h>
+#include <memfault/ports/zephyr/http.h>
+#include <memfault/core/data_packetizer.h>
+#include <memfault/core/trace_event.h>
+#include <memfault/core/custom_data_recording.h>
+#include <modem/nrf_modem_lib_trace.h>
+
+#include "modem_trace_memfault.h"
+#include "mosh_print.h"
+
+static bool memfault_cdr_source_registered;
+static const char *mimetypes = { MEMFAULT_CDR_BINARY };
+static sMemfaultCdrMetadata trace_recording_metadata = {
+	.start_time.type = kMemfaultCurrentTimeType_Unknown,
+	.mimetypes = &mimetypes,
```

Comment:
This will work as long as the number of `.num_mimetypes`  is 1

Reply:
I reverted back to how it was originally in the Memfault docs.